### DISCUSSION TITLE
如何在原创性说明页插入电子签名？

### DISCUSSION THREAD

**Discussion by WxxW2002** [sjtug/SJTUThesis#1037] [提问]
如何在原创性说明页插入电子签名以及如何编辑该页

**Comment by NemoYuan2008** [comment]
你好！目前的代码结构不适合直接编辑原创性说明页，一般是打印出来后，手写签字，再扫描成PDF后插入。

如有需要，你可以使用 PDF 编辑器（或者通过 word 模板）插入电子签名，将该页导出为 PDF 后，根据代码中注释的提示，使用 `\copyrightpage[文件名.pdf]` 插入，注意要在导言区使用 `\usepackage{pdfpages}`。

**Comment by echoJzjjy** [comment]
![image](https://github.com/user-attachments/assets/89abff08-1610-47be-a4c1-efb3b4eacc11)
![image](https://github.com/user-attachments/assets/e9bd1879-b494-4aad-ab83-9673bffe3f36)

我使用了\copyrightpage[文件名.pdf] 插入，并且在导言区使用 \usepackage{pdfpages}，但是渲染错误了

**Reply by LogCreative** [reply]
或许需要检查一下你插入的PDF是否完整？你可以直接点PDF下载按钮看一下真实文件，排除一下是不是网页无法渲染。

**Reply by echoJzjjy** [reply]
![image](https://github.com/user-attachments/assets/e53e3355-c7bb-4a2b-a291-ddaa7ff175b9)

看起来是没问题的

**Reply by LogCreative** [reply]
感觉文件损坏的可能性比较大，你要不手动用 Acrobat 或者 foxit 这类PDF编辑器替换下页面吧。
